src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -122,6 +143,125 @@ def get_shares(self) -> List[torch.Tensor]:
         """
         return self.shares
 
+    def get_ring_size(self) -> str:
+        """Ring size of tensor.
+
+        Returns:
+            ring_size(str): Returns ring_size of tensor in string.
+
+        It is typecasted to string as we cannot serialize 2**64
+        """
+        return str(self.ring_size)
+
+    def get_config(self) -> Dict:
+        """Config of tensor.
+
+        Returns:
+            config(Dict): returns config of the tensor as dict.
+        """
+        return dataclasses.asdict(self.config)
+
+    @staticmethod
+    def addmodprime(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
+        """Computes addition(x+y) modulo PRIME_NUMBER constant.
+
+        Args:
+            x(torch.Tensor): input tensor
+            y(torch.tensor): input tensor
+
+        Returns:
+            value(torch.Tensor): Result of the operation.
+
+        Raises:
+            ValueError : If either of the tensors datatype is not torch.uint8
+        """
+        if x.dtype != torch.uint8 or y.dtype != torch.uint8:
```

Comment:
Why only uint8, why not uint16 or uint32?

Reply:
Same for other ops.

Reply:
For PRIME_NUMBER order ring we tend to use small prime to decrease the share size to be communication efficient,Falcon mentions `37` , but for  practical implementation ,Falcon uses `67`. The prime_order ring is mapped to `torch.uint8` datatype.
https://github.com/OpenMined/SyMPC/blob/18a52383d0c72834c9825dc5d504d07c4e1ebd86/src/sympc/utils/mpc_utils.py#L16